### ISSUE TITLE
MSE LOSS

### ISSUE THREAD

**Issue by sonal-bansal** [yulunzhang/RDN#9]
Hi,
How much loss(MSE) did u obtained after your training got completed, I've trained my model for about 100 epochs and its ~100  ?
Moreover, I'm using RandomCrop of patches of LR and HR for training ...how will it effect my training?

Thank you

**Comment by yulunzhang** [comment]
Hi, 
I cannot remember the MSE loss exactly. Some of the log files are deleted. Anyway, if you use the training code to train a model with MSE, the validation and test performance should be similar or slightly lower than that using L1 loss.

For the usage of RandomCrop, I think it performs very similar as the original code dose. Thanks.
